src/cli/run.js

```diff
@@ -18,6 +20,11 @@ function run( args, options ) {
 	// Default to non-zero exit code to avoid false positives
 	process.exitCode = 1;
 
+	// Do not push content of .gitignore everytime when run is called
+	if ( IGNORED_GLOBS.length === 2 ) {
+		IGNORED_GLOBS.push.apply( IGNORED_GLOBS, utils.getIgnoreList( process.cwd() ) );
```

Comment:
Could we just do this when the module loads? E.g.:

```javascript
const IGNORED_GLOBS = [
  ".git",
  "node_modules"
].concat( utils.getIgnoreList( process.cwd() ) );
```

Then we can avoid hardcoding the length check.

Reply:
As I mentioned in the comment above the check, I was thinking if the `run` is called more than once we will be pushing `".git",  "node_modules"` multiple time. If it is not a valid case I am fine with doing this. Just that `IGNORED_GLOBS` need to be global const as it is used in `run.restart functions` to ignore files.

Reply:
And `push` is much faster than `concat`

Reply:
My suggestion is to do this outside of `run` (i.e., on line 9) so it'll only ever possibly run once.

And push vs. concat I don't particularly care about perf since we'll only be calling it once the different should be negligible.

Reply:
Sorry for the delayed update. I committed new changes as per review comments.